### PR TITLE
code(builtin) - add reverse function

### PR DESCRIPTION
**Pull request by PxyUp**
Here pull request on the https://github.com/expr-lang/expr/issues/552

### CODE REVIEW

File: builtin/builtin.go [hunk 1]
```diff
@@ -868,6 +868,40 @@ var Builtins = []*Function{
...
+
+		},
+		Validate: func(args []reflect.Type) (reflect.Type, error) {
+			if len(args) != 1 {
+				return anyType, fmt.Errorf("invalid number of arguments (expected 1, got %d)", len(args))
+			}
+			switch kind(args[0]) {
+			case reflect.Interface, reflect.Slice, reflect.Array:
+			default:
+				return anyType, fmt.Errorf("cannot reverse %s", args[0])
```
Comment (antonmedv) [inline builtin/builtin.go:900]: Let's move this return inside `case`.

**Review by antonmedv — CHANGES_REQUESTED** [review]
Please, add documentation to language-definition.md.

File: builtin/builtin.go (hunk 1 above)
Comment (PxyUp) [reply builtin/builtin.go:900]: What you mean? you still will need return from the function

Comment (antonmedv) [reply builtin/builtin.go:900]: Nope.

```go
// You can edit this code!
// Click here and start typing.
package main

import "fmt"

func main() {
	fmt.Println(foo(42))
}

func foo(i int) bool {
	switch i {
	case 42:
		return true
	default:
		return false
	}
}
```
https://go.dev/play/p/YtfTFf9GjZ9

**Pull request merged**